### DISCUSSION TITLE
i can upload a repo with time of  past time

### DISCUSSION THREAD

**Discussion by FortisLeo** [community/community#120296] [Repositories]
### Select Topic Area

Bug

### Body

I uploaded a repo from cli of android studio using my laptop which had time of 45 mins earlier of current time so when i uploaded repo and opened in my pc it was showing that i uploaded it 47 mins ago but i uploaded it 2 mins back, i think it's a pretty serious bug

**Comment by fdistasio** [comment]
This is a consequence of the fact that github allows you to make changes to the commit date even in the past. This is not a bug but is seen as a feature to allow developers more flexibility.

**Reply by fdistasio** [reply]
Does this answer your question?
